### PR TITLE
LogEncoder capnp implementation

### LINKED ISSUE #1990: Add capnp serialization to LogEncoder
See #1715

### PR DESCRIPTION
**Pull request by oxtopus**
Builds off of #1989 and fixes #1990

cc @scottpurdy

### CODE REVIEW

**Comment by rhyolight** [comment]
Travis failure looks suspicious. Restarted.

File: nupic/encoders/logenc.capnp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@0x94b5c39874f24431;
```
Comment (scottpurdy) [inline nupic/encoders/logenc.capnp]: I'd just call this `log.capnp` inline with previous naming choices.

File: tests/unit/nupic/encoders/logenc_test.py [hunk 2]
```diff
@@ -255,24 +259,66 @@ def testInitWithN(self):
...
-    expectedRadius = 0.552141792732 
+
+    expectedRadius = 0.552141792732
     expectedResolution = 0.110428358546
     self.assertAlmostEqual(le.encoder.radius, expectedRadius)
     self.assertAlmostEqual(le.encoder.resolution, expectedResolution)
 
 
+  def testReadWrite(self):
+    le = LogEncoder(w=5,
```
Comment (scottpurdy) [inline tests/unit/nupic/encoders/logenc_test.py]: `encoder1` and then `encoder2` for the deserialized one? `le` doesn't seem very descriptive.

**Comment by scottpurdy** [comment]
:+1:

**Pull request closed without merging**